driver/kubernetes/kubernetes.go

```diff
@@ -316,8 +316,15 @@ func (k *Driver) streamPodLogs(options metav1.ListOptions, out io.Writer, done c
 			// We successfully connected to the pod, so mark it as having streamed logs.
 			streamedLogs[podName] = true
```

Comment:
Wouldn't it be better to set this to true if and only if (all?) the logs have been streamed?